examples/Aggregator/Server/Program.cs

```diff
@@ -16,23 +16,50 @@
 
 #endregion
 
-using Microsoft.AspNetCore.Hosting;
-using Microsoft.Extensions.Hosting;
+using Count;
+using Greet;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Server;
 
-namespace Server
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddGrpc();
+builder.Services.AddSingleton<IncrementingCounter>();
+
+if (bool.TryParse(builder.Configuration["EnableOpenTelemetry"], out var enableOpenTelemetry) && enableOpenTelemetry)
```

Comment:
Is there no constant for the `EnableOpenTelemetry` key name?

Reply:
It's only used once.